# HOW TO PLAY

## WHAT'S THIS ABOUT? 

<span class="fuchsia bold">cy.br</span>/<span class="turquoise bold">dys.top.ia</span> is a roleplaying game set in a dystopic cyberpunk setting. 

You take the role of a *runner*: a mercenary, vigilante, hacker, anti-hero, or a combination of those. *Runners* usually fight against the oppression of the **MEGACORPS** that rule the world (although sometimes, they find themselves in need of working FOR the corps, albeit temporarily).

The setting of <span class="fuchsia bold">cy.br</span>/<span class="turquoise bold">dys.top.ia</span> is what is known as *"anti-cannon"* : It could have all the tropes of the cyberpunk genre, but nothing is written in stone. It is yours to shape however you like. Maybe your city is reminiscent of *The Sprawl*, or perhaps your game takes place in an orbital metropolis inside a *Stanford Torus*. Perhaps there's *replicants* roaming about, trying to free themselves from their corporate slavers, or maybe they are actively hunting or impersonating people and snatching their appeareances. Maybe in this world of yours, society has discovered how to digitally back up minds and humanity has become effectively immortal. Perhaps true AI exists and has its own devises, which brings lots of philosophical questions to civilization. You are free to pick or discard any trope or tenet of the cyberpunk genre to curate your own setting however you like.

## BUT... HOW DO YOU PLAY THIS?

A roleplaying game is a narrative activity, usually structured as a conversation between two or more people (or, if you are playing SOLO, with yourself or a written journal!). All the participants interpret a Character and describe their actions, desires and thoughts. Together they take turns to narrate a collaborative story. But this is not just a *make-believe excercise* or *improv session* : It is also a game, which means there are rules to follow, and sometimes whatever you decided your Character tries might fail. This is when the dice come into play: To adjudicate when something with fictional importance succeeds or fails. There's also a random factor implied with the Oracles, for when you need help to kickstart your creativity or when you don't know how to continue making stuff up. 

### PROMPTS AND TRIGGERS

This game has a list of **PROMPTS**, which are the main mechanical parts of the system. Prompts are comprised of a TRIGGER and some RESULTS, and some have options for you to make a choice.

The TRIGGER is something that happens in the fiction as the players narrate the story or their Characters' actions.

The RESULTS usually depend on the outcome of a dice roll, and are divided into <span class="turquoise bold">SUCCESS</span> and <span class="fuchsia bold">FAILURE</span> (with optional extras for <span class="turquoise bold">CRITICAL SUCCESS</span> or <span class="fuchsia bold">FAILURE</span>). 

> Later on you will learn how to roll the dice when prompted, and against what value you should compare to find out if you SUCCEED or FAIL at an action or narrative beat.

### LET FATE DECIDE

Since this game is designed for collaborative, improvised storytelling and on-the-spot world-building, you don't have a pre-defined story or sandbox scenario to run. Some people love to ad-lib and come up with every detail on their own, but this is rare and often very hard, unless that's your trade IRL. For the rest of us mortals, **ORACLES** come to help. 

**ORACLES** are basically random tables of content related to a topic, useful when you need to come up with details for something and you don't want to make it up yourself. In these situations, you just [QUERY THE FATES](/prompts/fate-prompts#query-the-fates) and <span class="turquoise bold">Spark an Idea</span> by using one of the many Oracles at hand in this web app. This helps move the story along without much delay, and sometimes surprises you with things or ideas that you might've not thought about. 

## WHERE DO YOU PLAY THIS?

Traditional RPGs evolved from tabletop wargames in the 1970's, which meant that players needed to gather around a table and talk to each other in person. Fortunately, advances in technology now make it possible to play remotely through any videoconference app and a web-browser. Many RPers love the classic way of playing together at the same place in a social gathering, and <span class="fuchsia bold">cy.br</span>/<span class="turquoise bold">dys.top.ia</span> obviously supports that mode, but it is also designed to be played online without much hassle.

In any case, each player will need some way to use this app on their own, be it with a mobile device capable of browsing the web, or with a hardwired computer. 

> For random rolling, it is recommended to use real dice (this game uses percentile dice, found in most hobby stores). But if you want to follow the trend established by this digital app, it is perfectly fine to use random generators (or virtual dice rollers) on your electronic device.

## HOW LONG DOES IT TAKE?

A session of <span class="fuchsia bold">cy.br</span>/<span class="turquoise bold">dys.top.ia</span> can take however you want, but the usual is 1 to 4 hours, depending on each participant desires, taste or schedule. It is also a game that can be played once, or many times. The same story (with the same characters and plot) can be continued over the course of many sessions, or you can do different stories each time you sit to play.

More on that later, on the [WHEN DOES THE GAME END?](/rules/how-to-play#when-does-the-game-end) section.

## THE THREE GAME MODES

This game supports three different game modes: <b>SOLO</b>, <b>COOP</b> or <b>GUIDED</b>.

> **SOLO** means it's just you, playing the game with a single Character, all by yourself. You also need to envision what happens with the environment and other characters around your protagonist. You will make the decisions for all of them, or use ORACLES to delegate those choices to fate. <br/>Some people like to talk out loud or record (or even stream!) themselves while playing. Others like to remain silent and envision every narrative beat to themselves, or write it down in a journal. Some even use this to write fiction. <br/>Experiment and use whatever method you like best. 

> **COOP** means there's 2 or more players each roleplaying a Character of their own. They make decisions for their own characters and also brainstorm and envision what happens with the environment and the Non-Player Characters (NPCs). There's noone in charge; everyone has the same duties and authority. <br/>Some groups like to let a single player take the reins for each scene and rotate the responsibility. Others like to discuss each decision and throw ideas or narrate as they come up with new details. <br/>There's no right or wrong here; Use whatever works best for your group.

> **GUIDED** means there's someone playing as **THE GUIDE**, while the rest of the participants play as single Characters. In other Role-playing games this role is known as the <i>Game-Master</i>, or the <i>Referee</i>. <br/>Usually, the GUIDE doesn't play as a Character, but instead describes (and has authoritative control over) the environment and all NPCs that are not the player Characters. This way, the rest of the players can lay back and leave the burden of improvising every story beat or twist to the more experienced GUIDE, and focus solely on their own Characters. <br/>Other groups prefer to have each Character be proactive, carrying the story by themselves and pursuing their own goals in an active manner, while the GUIDE reacts to those decisions and surprises them with unexpected situations. 

## THE MAIN GAME LOOP

The main part of the game is the narration. If playing **COOP** or **GUIDED**, it is a back-and-forth conversation between the participants. But the fiction gets interrupted by the mechanics in the form of **PROMPTS**. 

All players should be somewhat aware of the existing PROMPTS and its TRIGGERS. When the fiction being narrated matches one of these TRIGGERS, that PROMPT must be executed (most of the time, it will require a test from one of the player Characters). 

````
[...]
Player 1: Ne0n is combing the room in search for anything that might be useful. He's looking around for a while until he finds a datapad with the required info!
Player 2: Wait, that sounds like you are activating GATHER INFORMATION. Its trigger reads: "When you search for clues, conduct an investigation, analyze evidence, or do research...". 
Player 1: You are right! I need to roll the dice to see if I find that datapad or not. Let me see, it asks to TEST MY BRAIN. Ok, let's do this...  Oh no, I Failed!
Player 2: Oops! Now you need to follow the directives in the FAILURE section of the PROMPT. It says "your investigation unearths a dire threat or reveals an unwelcome truth that undermines your job.". That means that maybe you *did* find that datapad, but whatever it reveals is not what you expected, or exposes something bad for your mission.
Player 1: I see. What could that be? Let's QUERY THE FATES and use some ORACLES to find out...
[...]
````

To give some mechanical foundation and structure to every scene or the overall story, the game encourages you to wrap anything in a **CHALLENGE**. 

A **CHALLENGE** is kind of like a CLOCK, or a PROGRESS BAR towards an end. Some PROMPTS will instruct you to [MAKE PROGRESS](/prompts/challenge-prompts#make-progress) in the current CHALLENGE, and when you eventually [FULFILL A CHALLENGE](/prompts/challenge-prompts#fulfill-a-challenge) you earn XP to later [IMPROVE YOURSELF](/prompts/fate-prompts#improve-yourself). 

> **CHALLENGES** can be nested inside of each other, and depending on your actions, you might [MAKE PROGRESS](/prompts/challenge-prompts#make-progress) in the innermost (and currently focused) one, on another from a higher level, or many at once.

You will learn more about **CHALLENGES** later, in [its own section](/rules/how-to-play#challenges).

### KNOW YOUR CHARACTER

Your Character has several values and properties that might change during the game. Most of them are used or referenced by other mechanics in the rules.

#### STATS

Your **STATS** are one of the most important ones: You will use them frequently when doing tests as instructed by the **PROMPTS**. They range from 1 to 99, the higher the better (because when testing them, you want to roll a **d100** under that value).

* Your <span class="stat">BRAIN</span> is a value that represents your *cunning and knowledge*. 
* Your <span class="stat">CHROME</span> depicts your character's *strength and endurance*. 
* Your <span class="stat">EDGE</span> portrays your character's *speed and precision*. 
* Your <span class="stat">FLASH</span> describes your character's *personality and empathy*. 
* Your <span class="stat">SHADE</span> denotes your character's *deception and stealth*. 

Experienced *Runners* will eventually have high values in most of their **STATS**, making them get <span class="turquoise bold">SUCCESS</span> results more often.

#### CHARACTER STATE

This is a toggle between two possible states: <span class="turquoise bold">IN CONTROL</span> or <span class="fuchsia bold">IN A BAD SPOT</span>. You will always be in one of them, no matter the context.

This state has diegetic significance and changes your fictional positioning: If you are <span class="turquoise bold">IN CONTROL</span>, envision or describe your actions like if you have the upper hand or you are in your element. When you are <span class="fuchsia bold">IN A BAD SPOT</span> you are at disadvantage, stumbling or trying to regain your footing (describe your actions accordingly).

The state also has mechanical importance: Many of the **PROMPTS** have different outcomes whether your are <span class="turquoise bold">IN CONTROL</span> or <span class="fuchsia bold">IN A BAD SPOT</span>, and they can also alter this state depending on the result of your dice rolls.

#### GRIT, PYSCHE, GEAR

GRIT, PYSCHE and GEAR are tracks that go from 0 to 5, representing your current state in different aspects. 5 is the maximum and means a good condition. The closer to 0, the worst that axis of your Character looks like.

* **GRIT** is your physical resistance or health quality. 5 means you are in perfect condition; 0 means you are hurt, tired, maimed, etc.
* **PSYCHE** is your mental state, or your current psychological condition. 5 means you are at your best, while 0 might mean you are overly stressed, terrified, traumatized, etc.
* **GEAR** is an abstraction of your tools and resources. You will spend it to produce useful items or to "buy" stuff and pay for services. 5 means you are well-equipped or have access to assets or funds; 0 means you are out of supplies or don't have enough cash to do anything.

#### ANGLE

Each Character wants something, or has a long-term objective of some sort. This is represented by your **ANGLE**: Think of it as a long-term **CHALLENGE** or final destination. It begins with a value of 1, and has an *EPIC* length, which means it will take a while before you can even attempt to fulfill it with good chances of success.

The Character sheet has some broad categories to help you think about your ANGLE. Pick one and elaborate on what that means in your NOTES section.

This is your Characters' *END GAME*: You are working towards your ANGLE; when it's time to [RETIRE](/prompts/fate-prompts#retire), you will test against its value.

#### TRAITS

**TRAITS** are specific boons your Character has, that give mechanical benefits to certain situations, but also imply your profession, knacks and tools of the trade. This is how you personalize your Character and differentiate them from others.

You can toggle each **TRAIT** you earn in [its own section](/character-traits) of the app. They usually grant a <span class="boost">BOOST</span> to specific **PROMPTS** or situations, and also define whatever gear or cyberware you possess, or **CONTACTS** you might be acquainted with. 

## DICE AND TESTS

This game uses *percentile dice*. These are 10-sided dice that have values of tens on each face, ranging from 00 to 90. They are often used in combination with regular ten-sided dice (with values ranging from 0 to 9), to generate a result between 1 (when the dice show 00 + 1) and 100 (00 + 0 showing on the dice).

When instructed to roll the dice, you will always want to get the lowest result possible, because you will have to compare against a value in your Character sheet or in a **CHALLENGE**. This means that the percentage of the probability is very intuitive: For example, if you need to test against a value of 70, you instantly know that you have a 70% chance of success. If you roll and get a result of 71 or higher, you have failed. Since values will never be lower than 1 or higher than 99, there is always a 1% chance of succeeding or failing, no matter how many <span class="boost">BOOSTS</span> or <span class="glitch" data-text="GLITCHES">GLITCHES</span> you have for that roll.

Most of the time, you will test against one of your **STATS**. If a PROMPT instructs you to "test your <span class="stat">CHROME</span>" and your <span class="stat">CHROME</span> has a value of 65, any roll result between 1 and 65 means <span class="turquoise bold">SUCCESS</span>, while 66 or greater means <span class="fuchsia bold">FAILURE</span>.  

In some circumstances you will need to test against a **CHALLENGE** value. The procedure is the same: If the **CHALLENGE** value is 54, when you roll the dice, any result between 1 and 54 means <span class="turquoise bold">SUCCESS</span>, while 55 or greater means <span class="fuchsia bold">FAILURE</span>. This is why you will want to get that **CHALLENGE** value as high as possible before attempting to [FULFILL A CHALLENGE](/prompts/challenge-prompts#fulfill-a-challenge) !

### CRITICALS

A **CRITICAL** is a roll result that greatly exacerbates the outcome of the desired action. It could be a <span class="turquoise bold">CRITICAL SUCCESS</span> or a <span class="fuchsia bold">CRITICAL FAILURE</span>.

When you roll the dice, look at both tens and units results: A match means you've gotten a **CRITICAL**. On top of that, a 01 is always a <span class="turquoise bold">CRITICAL SUCCESS</span>, and a 100 is always a <span class="fuchsia bold">CRITICAL FAILURE</span>. 

```
A Player needs to test against their Character's FLASH. They have a FLASH value of 60. They roll the dice and...

Example 1: 
The dice come up as 30 and 5, for a 35. This is under their FLASH value of 60, wich means a regular SUCCESS.

Example 2:
The dice come up as 70 and 1, for a 71. Since this is over 60, the player got a regular FAILURE.

Example 3:
The dice come up as 50 and 5, for a 55. This is a match, which means a CRITICAL! and since it's under 60, it produces a CRITICAL SUCCESS.

Example 4:
The dice come up as 60 and 6, for a 66. This is also a match, but since it's over 60, it provokes a CRITICAL FAILURE.

Example 5:
The dice come up as a 00 and 1, for a 1. This is a CRITICAL SUCCESS, no matter what value the Player was testing against.

Example 6:
The dice come up as a 00 and 0, for a 100. This is a CRITICAL FAILURE, no matter what value the Player was testing against.

```

> By looking at the examples above and doing some quick math you might have noticed that the chance of getting a **CRITICAL** is always the same: There's 10 possible matches of tens and units in a d100 roll (11, 22, 33, etc), which combined with the 01 (that is always a critical success) give a constant 11% chance of rolling a **CRITICAL**. What does change is the possibility of it being a <span class="turquoise bold">CRITICAL SUCCESS</span> or <span class="fuchsia bold">FAILURE</span>, and that depends entirely on the value you test against (meaning that the higher it is, the most likely that the **CRITICAL** is going to be a <span class="turquoise bold">SUCCESS</span>).

When you get a **CRITICAL**, most **PROMPTS** will have a different or extra outcome that you must abide to. But you are also encouraged to envision whatever fictional result you would describe as an amplified version (for good or ill !).

### BOOSTS AND GLITCHES 

These are mechanical concepts that alter your dice rolls. 

* A <span class="boost">BOOST</span> is and advantage or edge you get for a test, increasing your odds.
* A <span class="glitch" data-text="GLITCH">GLITCH</span> is a disadvantage or bad condition that lowers your chances of success.

<span class="boost">BOOSTS</span> or <span class="glitch" data-text="GLITCHES">GLITCHES</span> will be granted to you by <span class="bold">PROMPT</span> instructions, your <span class="bold">TRAITS</span>, other Characters helping you, etc. A single test might be affected by more than one <span class="boost">BOOST</span> or <span class="glitch" data-text="GLITCH">GLITCH</span>.

<br/>
<br/>
<span class="boost">BOOSTS</span> and <span class="glitch" data-text="GLITCHES">GLITCHES</span> cancel each other 1 to 1. If you have the same amount of <span class="boost">BOOSTS</span> and <span class="glitch" data-text="GLITCHES">GLITCHES</span>, they cancel each other and you end up with a regular d100 roll.

<br/>
<br/>
If, instead, you end up with one or more <span class="boost">BOOSTS</span> or <span class="glitch" data-text="GLITCHES">GLITCHES</span>, you must roll an extra d% (the tens die) for each <span class="boost">BOOST</span> or <span class="glitch" data-text="GLITCH">GLITCH</span>: If you had <span class="boost">BOOSTS</span>, you keep the best one of these. If there were <span class="glitch" data-text="GLITCHES">GLITCHES</span>, you keep the worst one.

<br/>
<br/>

> **What does "the best one" or "the worst one" mean?** <br/>Usually, the best one is the lowest result, and the worst one is the highest. But when there's a match and CRITICALS come into play, these have priority for what's best or worst.

You still only roll 1 units die, no matter how many <span class="boost">BOOSTS</span> or <span class="glitch" data-text="GLITCHES">GLITCHES</span> you have for that test. This is important because by keeping the units number fixed, the odds of getting a **CRITICAL** greatly increase for each tens' dice added by <span class="boost">BOOSTS</span> or <span class="glitch" data-text="GLITCHES">GLITCHES</span>.

```
Example 1:
You have a BOOST for your roll against a value of 42. You must roll two tens' dice (and keep the best), and a single units dice.

> You get a result of 10, 60 and 2: The best combination would be 12, so you must take that result.

Example 2: 
You have a BOOST for your roll against a value of 42. You must roll two tens' dice (and keep the best), and a single units dice.

> You get a result of 10, 30 and 3: The lowest combination would be 13, but since 33 is also under 42 and is also a CRITICAL, you must take that result.

Example 3:
You have two GLITCHES for your roll against a value of 36. You must roll three tens' dice (and keep the worst), and a single units dice.

> You get a result of 10, 40, 50 and 4: The highest combination would be 54, but since 44 is also over 36 and is also a CRITICAL, you must take that result.

```

## CHALLENGES

A **CHALLENGE** is a key mechanical procedure for this game. Whenever you need to do something that cannot be accomplished by a single action (or **PROMPT** resolution), you could represent it as a CHALLENGE. When there is a complex endeavor that could be broken into several tasks, you most certainly could wrap it in a CHALLENGE. 

CHALLENGES have a value that represents its level of completion. It starts at 1. Depending on its **RANK**, you will increase a certain fixed amount every time you [MAKE PROGRESS](/prompts/challenge-prompts#make-progress) on that CHALLENGE. The maximum value is 99 (which means that, even when maxed up, you still have a 1% chance of failure).

> **CHALLENGE RANKS**<br/><br/> 
**SHORT**: Progress by increments of +30% (it takes 4 progress to maximize chances). It nets 2 XP when fulfilled.<br/><br/>
**MEDIUM**: Progress by increments of +20% (it takes 5 progress to maximize chances). It nets 5 XP when fulfilled.<br/><br/>
**LONG**: Progress by increments of +10% (it takes 10 progress to maximize chances). It nets 10 XP when fulfilled.<br/><br/>
**EXTREME**: Progress by increments of +5% (it takes 20 progress to maximize chances). It nets 20 XP when fulfilled.<br/><br/>
**EPIC**: Progress by increments of +2% (it takes 50 progress to maximize chances). It nets 50 XP when fulfilled.<br/>

At any time, you can decide to attempt to [FULFILL THAT CHALLENGE](/prompts/challenge-prompts#fulfill-a-challenge), testing against its value. Obviously, the greater that value, the better chances you will have at this test.

A **CHALLENGE** works as a *pacing mechanism*: You decide its RANK (which defines its length), directly impacting on how many actions you will have to do to [MAKE PROGRESS](/prompts/challenge-prompts#make-progress) and increase its value to a number high enough that you can attempt to fulfill it. When playing in **GUIDED** mode, it's usually the **GUIDE** who decides the CHALLENGE RANK.

CHALLENGES also work as objectives or *narrative roadmaps*: When beginning a mission, you should [ACCEPT A CHALLENGE](/prompts/challenge-prompts#accept-a-challenge). Then, during said mission, each scene could be wrapped in another CHALLENGE ([ACCEPT A CHALLENGE](/prompts/challenge-prompts#accept-a-challenge) again!). Inside those scenes, smaller tasks that demand focus could be represented as CHALLENGES. As a rule of thumb, the "outermost" CHALLENGES should be of higher RANK than the "innermost" ones, and it's common sense that you should fulfill the innermost challenges before the bigger ones that contain them.

Players are encouraged to accept and fulfill CHALLENGES constantly by the reward in XP they provide. But [FAILING A CHALLENGE](/prompts/challenge-prompts#fail-a-challenge) is not necessarily a bad thing: Sometimes it's better to abandon one if it turns out gameplay is dragging too much. Specially if you are juggling many CHALLENGES at once, abandoning one won't hurt much.

Sometimes CHALLENGES end naturally as part of the fiction. You can decide not to roll and fulfill it automatically or declare it abandoned (but avoid [PAYING THE CONSEQUENCES](/prompts/fate-prompts#pay-the-consequences) of a FAILED CHALLENGE). If you do this, do so sparingly as it goes against the main mechanics of the rules system (*PROMPT directives must be abided as much as possible!*).

Other times, CHALLENGES end up being much longer than you envisioned, and you realize that you should have made it of a lower RANK. It's okay, you can reduce its RANK one or more levels until you get the hang of it. Doing so will make it easier (or faster) to get it to a coherent value to fulfill, but it will also penalize you with a lower XP reward.


## CHARACTER CREATION

Creating your Character is a process that involves filling the corresponding fields and inputs as you envision your protagonist.

Some people like to take as much time as needed to come up with a perfect and well-defined character; Others like to outline the mere basics and find out the rest of the details as they play the game.

If you are playing COOP or GUIDED, it is recommended that all players involved create their Characters together, talking about the details and thinking of the relations of each protagonist in the group.

### STEP-BY-STEP CHARACTER CREATION PROCESS 

For this process you will mostly be switching between the [STATS](/character-stats) and [TRAITS](/character-traits) sections of the web app.

1. Set your **XP current / total** to **0/0**.
2. Set your state to <span class="turquoise bold">IN CONTROL</span>.
3. Distribute these values however you like among your **STATS**: 60, 50, 40, 30, 20. You will want to put the highest values into the STATS that you think your Character should be strongest at, and the lowest at their weak spots.
4. Set your **GRIT**, **PYSCHE** and **GEAR** to 5.
5. Choose an **ANGLE** from the options provided. The cues provided in the Character sheet are a starting point; elaborate on your ANGLE and write it down on the NOTES section. 
6. Choose 3 **TRAITS** from the SKILLS AND BACKGROUND*, *GEAR AND CYBERWARE* and *CONTACTS* sections. Later on you will be able to gain new ones by spending XP.
7. Give your Character a name or callsign, and envision their looks and basic personality (use [Character Oracles](/oracles/character-oracles#general-characters) if neeeded). If playing with others, introduce these details to everyone before the story begins.

> **A NOTE ON "POWER LEVEL"**<br/><br/>
Starting Characters are relatively strong in some few areas (depending on **STAT** and **TRAIT** combinations) but decidedly weak in most others. When playing as a group, it is recommended that everyone fills a certain niche, so that most of the group needs are somewhat covered by a specialist.<br/><br/>
When playing **SOLO** (or **GUIDED** but with only one Player Character), the game might result too hard (that means: there will be a great chance of <span class="fuchsia bold">FAILURE</span> in most **PROMPTS** and tests). Some people like this sort of challenge. But if you find yourself struggling too much or consider that the amount of <span class="fuchsia bold">FAILURES</span> don't line up with how you envisioned your story, consider using the **LONE WOLF** mode:<br/>
The Character starts with 50XP that you can immediately use to [IMPROVE YOURSELF](/prompts/fate-prompts#improve-yourself) and increase some of their **STATS** or buy some extra **TRAITS**.


## HOW TO START A GAME

TODO

## WHEN DOES THE GAME END?

TODO
